src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -51,3 +52,15 @@ class OrchestratorService(RESTController):
     def list(self, hostname=None):
         orch = OrchClient.instance()
         return [service.to_json() for service in orch.services.list(None, None, hostname)]
+
+
+@ApiController('/orchestrator/osd', Scope.OSD)
+class OrchestratorOsd(RESTController):
+
+    @raise_if_no_orchestrator
+    def create(self, drive_group, all_hosts=None):
+        orch = OrchClient.instance()
+        try:
+            orch.osds.create(DriveGroupSpec.from_json(drive_group), all_hosts)
+        except (ValueError, DriveGroupValidationError):
```

Comment:
The DriveGroupValidationError might contain some detailed infos.

Reply:
Thanks, fixed in https://github.com/ceph/ceph/pull/30921/commits/b35391abafaa809521d659459ec163656c1dc26d#diff-84358f17f642ed20b036d835ffd6b476R66-R67